### PR TITLE
[PARTOPS-1028] Add loki status info to User Feedback doc

### PR DESCRIPTION
**Pull request by michellechu77**
[Loki Status project brief](https://coda.io/d/Team-Developer-Platform_di0MgBhlCWf/One-Pager-Loki-Issue-Statuses_su4o4#_lutLR)

Adding information about the new loki statues on the existing "Respond to user feedback and bugs" platform doc.

### CODE REVIEW

**Comment by justin-lee-zapier** [comment]
When you're ready for @team-platform-support to review this, please head over to this [Slack thread](https://zapier.slack.com/archives/C02PCUNBUG7/p1712864588247789) and give us a `:review-pr:` emoji.

File: docs/_manage/integration-performance/user-feedback.md [hunk 1]
```diff
@@ -51,9 +53,7 @@ Any reported issue with no updates from you, Zapier Support, or end users for a
 
 ![Changelog Form with Issues](https://cdn.zappy.app/8afba17ecfb25faf6b87597e2cd54387.png)
 
-- You can also request issues to be closed manually by selecting one in the "Bugs & Feature Requests" tab of the developer platform, and leaving a comment to our Support team that the issue has been resolved. Provide info on what version number the fix is on, how to test the updates, and any other helpful callouts.
-
-- Once an issue is reviewed, it will either be closed or responded to with follow-up questions and feedback.
+- You can also request issues to be closed manually by updating their statuses to `Done`. Make sure updates are deployed and promoted to the lastest version of your integration before marking issues as `Done`, as it will be reviewed by a Zapier support member. See the [`Utilizing issue statuses` section](https://platform.zapier.com/manage/user-feedback#utilizing-issue-statuses) below for more details and best practices.
```
Comment (marinahand) [inline docs/_manage/integration-performance/user-feedback.md:56]: lastest - typo :)

Comment (marinahand) [reply docs/_manage/integration-performance/user-feedback.md:56]: I suggest editing to "Make sure updates are deployed, version promoted and users migrated if possible before marking issues..." as we often see devs forget to migrate.

File: docs/_manage/integration-performance/user-feedback.md [hunk 2]
```diff
@@ -99,4 +99,26 @@ Any reported issue with no updates from you, Zapier Support, or end users for a
...
+* _Rejected_: After review, Zapier deemed the issue resolution not complete.
+
+Once an issue is resolved, update the issue status to `Done`. You will be prompted to select one of the following issue resolutions:
+
+* _Resolved_: Work is completed and updates have been promoted.
+* _Won’t Do_: This issue won't be actioned (i.e. not on the roadmap, not technically possible, etc.). A comment explaining why is required.
+* _Duplicate_: The issue is a duplicate of another issue. Please note the duplicate issue in the required comment.
+* _Cannot Reproduce_: All attempts at reproducing this issue failed.
+
+All issues marked `Done` are reviewed by the Zapier team for accuracy and completion. Any necessary explanation or feedback will be provided via comments on respective the issues after the internal review.
```
Comment (marinahand) [inline docs/_manage/integration-performance/user-feedback.md:124]: "will be provided via comments on respective issues after the internal review. Please respond with questions or concerns on respective issues to keep things clear. "

Comment (michellechu77) [reply docs/_manage/integration-performance/user-feedback.md:124]: Thanks Marina!!

**Review by marinahand — DISMISSED** [review]

File: docs/_manage/integration-performance/user-feedback.md [hunk 3]
```diff
@@ -99,4 +99,28 @@ Any reported issue with no updates from you, Zapier Support, or end users for a
...
-- Share access to the insights by [inviting collaborators](https://platform.zapier.com/manage/invite-team-member) to the integration without giving them permissions to make changes to it.
\ No newline at end of file
+- Share access to the insights by [inviting collaborators](https://platform.zapier.com/manage/invite-team-member) to the integration without giving them permissions to make changes to it.
+
+## Utilizing issue statuses
+
+Maintaining accurate issue statuses during regular triage and review of your integration's issues is critical for keeping your integration healthy. Not only do statuses help your team track progress and build roadmaps, but they also signal your engagement and prompt reviews for fixed issues to the Zapier team.
+
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="600" height="500" src="https://on.driveway.app/guides/zEVVdGE/embed" allow="clipboard-write" allowfullscreen></iframe>
+
```
Comment (redbeardaz) [inline docs/_manage/integration-performance/user-feedback.md:109]: I might add some helptext under the iframe like (Enter fullscreen for a better experience). 

I know it was weird looking just in the normal size and it took me a minute to figure out there was a fullscreen option. 

Also feel free to disregard if this is a me issue.

**Review by redbeardaz — APPROVED** [review]

**Pull request merged**